bin/alluxio-mount.sh

```diff
@@ -99,10 +99,18 @@ function mount_ramfs_linux() {
     reduce alluxio.worker.memory.size in alluxio-site.properties" >&2
     exit 1
   fi
+  local mount_cmd="mount"
```

Comment:
seeing as the mount / unmount / chmod command is only used in one place, I would prefer to avoid defining a variable for it and instead do:

```
if [[ "${1}" == "SudoMount" ]]; then
  sudo umount -f ${TIER_PATH}
else
  umount -f ${TIER_PATH}
fi
```

and similarly for the `mount` and `chmod`